.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,22 @@
+# Add to: .github/workflows/spellcheck.yml
+name: Typo CI
+
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
There's no master branch, this is kind of pointless. What you should ignore is the gh-pages branch, because it'd already be checked from main.

Reply:
sorry it was a copypaste